Inference detect.py does not support .asf video format

### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar feature requests.


### Description

The ".asf" file format is not recognized. 
Running: `python3 detect.py --source "myvideofile.asf"` results in the following output:

`AssertionError: No images or videos found in /mnt/rm-data/Smart 1 04.10.21 06.00-08.00/9 - 2021-10-04 06-37-46-371.asf. Supported formats are:
images: ['bmp', 'dng', 'jpeg', 'jpg', 'mpo', 'png', 'tif', 'tiff', 'webp']
videos: ['avi', 'gif', 'm4v', 'mkv', 'mov', 'mp4', 'mpeg', 'mpg', 'wmv']`


Simply adding 'asf' to the utils/datasets.py video list solves my issue and the file gets processed without any further changes to the code.

### Use case

Everyone with .asf files could benefit.

### Additional

_No response_

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

--- a/utils/datasets.py
+++ b/utils/datasets.py
@@ -34,7 +34,7 @@ from utils.torch_utils import torch_distributed_zero_first
 # Parameters
 HELP_URL = 'https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data'
 IMG_FORMATS = ['bmp', 'dng', 'jpeg', 'jpg', 'mpo', 'png', 'tif', 'tiff', 'webp']  # include image suffixes
-VID_FORMATS = ['avi', 'gif', 'm4v', 'mkv', 'mov', 'mp4', 'mpeg', 'mpg', 'wmv']  # include video suffixes
+VID_FORMATS = ['asf', 'avi', 'gif', 'm4v', 'mkv', 'mov', 'mp4', 'mpeg', 'mpg', 'wmv']  # include video suffixes
 DEVICE_COUNT = max(torch.cuda.device_count(), 1)
 
 # Get orientation exif tag

